src/SDKs/Batch/DataPlane/Azure.Batch/CloudJob.cs

```diff
@@ -573,7 +573,39 @@ public void Refresh(DetailLevel detailLevel = null, IEnumerable<BatchClientBehav
             asyncTask.WaitAndUnaggregateException(this.CustomBehaviors, additionalBehaviors);
         }
 
-#endregion // CloudJob
+        /// <summary>
+        /// Gets the task counts for the job.
+        /// </summary>
+        /// <param name="detailLevel">A <see cref="DetailLevel"/> used for controlling which properties are retrieved from the service.</param>
+        /// <param name="additionalBehaviors">A collection of <see cref="BatchClientBehavior"/> instances that are applied to the Batch service request after the <see cref="CustomBehaviors"/> and <paramref name="detailLevel"/>.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> for controlling the lifetime of the asynchronous operation.</param>
+        /// <returns>A <see cref="TaskCounts"/> object containing the task counts for the job.</returns>
```

Comment:
Do we usually have a `remarks` on async ops pointing to the blocking equivalent?  I thought we used to.  (It's fine if we've ditched it.)